rispy/utils.py

```diff
@@ -0,0 +1,37 @@
+"""Miscellaneous functions."""
+
+from typing import Dict, List
+from copy import deepcopy
+
+from .config import TYPE_OF_REFERENCE_MAPPING
+
+
+def invert_dictionary(mapping: Dict) -> Dict:
+    """Invert the keys and values of a dictionary."""
+    remap = {v: k for k, v in mapping.items()}
+    if len(remap) != len(mapping):
+        raise ValueError("Dictionary cannot be inverted; some values were not unique")
+    return remap
+
+
+def pretty_reference_types(
+    reference_list: List,
```

Comment:
this should be a `List[Dict]` right?  Just so it's easier to understand what this is